### DISCUSSION TITLE
istoreos旁路由下openclash无法访问外网

### DISCUSSION THREAD

**Discussion by xxylyf** [vernesong/OpenClash#3402] [General]
用passwall翻墙，访问外网正常，其它不动，关闭passwall 启用openclash后，订阅链接正常，主界面运行状态正常，就是无法访问外网。有没有大哥知道怎么回事。

**Comment by baigeixiangcai** [comment]
> 用passwall翻墙，访问外网正常，其它不动，关闭passwall 启用openclash后，订阅链接正常，主界面运行状态正常，就是无法访问外网。有没有大哥知道怎么回事。

一样的情况，有人知道怎么回事吗
